README.md

```diff
@@ -23,7 +23,7 @@ fn main() {
         List::new(
             vec![
                 Edn::Symbol("sym".to_string()),
-                Edn::Double(1.2),
+                Edn::Double(1.2.into()),
```

Comment:
I think `into` is better than `Double(1.2)`. Also `Double` could be set as private so people don't use this option and don't know the inner logic.

Reply:
We can do this, I only left this public because of the other variants like `Set`, `Map`, etc, so I thought it could be intuitive for the user to just do the same for Doubles, since just adding a f64 won't work.

Reply:
In the future we could make a PR to try this with with destructured tuples `(sign, mantissa, exponent)`

Reply:
Oh, OK.
We need to filter what is public and what is private in this crate. For now almost everything is public kkkkk

Reply:
For now, we will let it public, then later we will filtrate what is not necessary.